cpucounters.cpp

```diff
@@ -6281,20 +6281,22 @@ void ServerPCICFGUncore::doMemTest(const ServerPCICFGUncore::MemTestParam & para
         }
 }
 
+#if defined(__linux__) || defined(_MSC_VER)
 void ServerPCICFGUncore::cleanupMemTest(const ServerPCICFGUncore::MemTestParam & param)
 {
     const auto & memBufferBlockSize = param.first;
     const auto & memBuffers = param.second;
     for (auto b : memBuffers)
     {
-#ifdef __linux__
+#if defined(__linux__)
         munmap(b, memBufferBlockSize);
 #elif defined(_MSC_VER)
         VirtualFree(b, memBufferBlockSize, MEM_RELEASE);
 #else
```

Comment:
possibly if you write 
```
memBuffers; // avoid the unused variable warning
memBufferBlockSize; avoid the unused variable warning
```
here it will help to avoid the warning